test/test.js

```diff
@@ -149,6 +153,20 @@ describe('clipboard-copy element', function () {
       const text = await whenCopied
       assert.equal(text, 'I am a link')
     })
+
+    it('does not copy when disabled', async function () {
+      const target = document.createElement('div')
+      target.innerHTML = 'Hello world!'
+      target.id = 'copy-target'
+      document.body.append(target)
+
+      const button = document.querySelector('clipboard-copy')
+      button.setAttribute('aria-disabled', 'true')
+      button.click()
+
+      const text = await whenCopied
+      assert.equal(null, text)
```

Comment:
Something like this should work:

```suggestion
      let fired = false;
      document.addEventListener('clipboard-copy', () => { fired = true }, { once: true })
      await new Promise(setTimeout)
      assert.equal(fired, false);
      assert.equal(null, text)
```